protocol/x/clob/keeper/liquidations.go

```diff
@@ -222,34 +224,49 @@ func (k Keeper) PlacePerpetualLiquidation(
 
 	// TODO(DEC-1323): Potentially allow liquidating the same perpetual + subaccount
 	// multiple times in a block.
+	perpetualId := liquidationOrder.MustGetLiquidatedPerpetualId()
 	k.MustUpdateSubaccountPerpetualLiquidated(
 		ctx,
 		liquidationOrder.GetSubaccountId(),
-		liquidationOrder.MustGetLiquidatedPerpetualId(),
+		perpetualId,
 	)
 
-	telemetry.IncrCounter(
-		1,
-		metrics.Liquidations,
-		metrics.PlacePerpetualLiquidation,
-		metrics.Count,
-	)
+	labels := []gometrics.Label{
+		metrics.GetLabelForIntValue(metrics.PerpetualId, int(perpetualId)),
+		metrics.GetLabelForBoolValue(metrics.IsBuy, liquidationOrder.IsBuy()),
+	}
 
+	// Stat the number of liquidation orders placed.
```

Comment:
Should we stat this outside of `PlacePerpetualLiquidation`, maybe [here](https://github.com/dydxprotocol/v4-chain/blob/d9fc60643fabfe028afd21e26a0820b3b042d070/protocol/x/clob/keeper/liquidations.go#L80) in `LiquidateSubaccountsAgainstOrderbook`?

Reply:
I think it's more appropriate here since this is the entry point for each liquidation?

Reply:
Yeah basically was hoping to also have a stat for liquidation volume per block (along with this stat)

For example, if you wanted to check how many liquidation orders were placed after a large price update

Reply:
I think some quick math will get us something pretty close. e.g. aggregation over 1 minute period divided by block time.